test/library/standard/Set/setAddLocalRemote.chpl

```diff
@@ -0,0 +1,55 @@
+use Set;
+
+class C { var x = 0; }
+
+record r {
+  var x = new shared C();
+  proc init() { writeln('default init'); }
+  proc init=(other: r) { writeln('init='); }
+  proc deinit() { writeln('deinit'); }
+}
+
+operator r.=(ref lhs: r, rhs: r) { writeln('assign'); }
+
+proc create(type T): T where isPrimitiveType(T) {
+  return 0:T;
+}
+
+proc create(type T) {
+  return new T();
+}
+
+proc testAddToSetRemote(type T) {
+  writeln('Adding to set of ' + T:string + ' remotely');
+  var s: set(T);
+  on Locales[1] {
+    var x = create(T);
+    s.add(x);
+  }
+  writeln(s);
+}
```

Comment:
Can you add another version where the element and the set are on the same locale, but addition happens elsewhere?


```chapel
var s: set(T);
var x = create(T);
on Locales[1] {
  s.add(x);
}
```

Reply:
Will do.